plugins/gatsby-theme-iterative-docs/src/components/Documentation/Markdown/Admonition/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import cn from 'classnames'
+import * as styles from './styles.module.css'
+
+const Admonition: React.FC<{
+  title?: string
+  type?: 'info' | 'tip' | 'warn'
+  icon?:
+    | 'tip'
+    | 'info'
+    | 'warn'
+    | 'fire'
+    | 'exclamation'
+    | 'lady_beetle'
+    | 'bug'
+}> = ({ title, type = 'info', children, icon = type }) => {
+  const icons = {
+    tip: '💡',
+    info: 'ℹ️',
+    warn: '⚠️',
+    fire: '🔥',
+    exclamation: '❗',
+    lady_beetle: '🐞',
+    bug: '🐛'
+  }
```

Comment:
I think it would be better to have a default fallback to a particular type and don't show the icon if not on the list.

<img width="421" alt="Screen Shot 2022-03-23 at 19 57 45" src="https://user-images.githubusercontent.com/20840228/159720836-1b15ea71-e407-492b-856b-d6b9120fd0f5.png">
 
<img width="841" alt="Screen Shot 2022-03-23 at 19 57 35" src="https://user-images.githubusercontent.com/20840228/159720820-3d203b62-aa5a-4a9e-9e6e-774aa32a0896.png">

Reply:
Good idea! Update the code If the type doesn't exist, it defaults to info color and if the provided icon doesn't exist, it doesn't show any icon!